### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by corecoding** [brianvoe/slim-select#503] [Ideas]
First off, I wanted to say thank you for creating slim-select. I was looking to drop jQuery and Select2, and this was the answer. There are a lot of type ahead drop down projects, but none had all of the features/methods/events that I needed.

My question/feature request is dark mode support. I am using bootstrap 5, and when I use their native dark mode, Slim Select stays white and the text field has a white foreground color as well (white on white) which is invisible.

Just curious on your thoughts on supporting dark mode (or if I am missing documentation somewhere).

**Comment by brianvoe** [comment]
Sure submit a pr
